include/odp/api/spec/queue_types.h

```diff
@@ -220,7 +222,7 @@ typedef struct odp_queue_capability_t {
 
 		} waitfree;
 
-	} sched;
+	} ODP_DEPRECATE(sched);
```

Comment:
This commit cannot move due to deprecation: "api: queue, schedule: move scheduled queue capabilities to sched" (64e4daf842cca70d0160abf36316ee70c74f67cf)